.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v[0-9]+(\.[0-9]+)*'
+jobs:
+  publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '14' ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          target: wasm32-unknown-unknown
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+          registry-url: 'https://registry.npmjs.org'
+      - name: Prepare publish
+        run: npm ci
+      - name: Publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH }}
+        run: npm publish --access public
```

Comment:
Do you have thoughts on using `wasm-pack publish`? I've not actually tried it.

Reply:
I wasnt aware of that, I will take a look https://github.com/rustwasm/wasm-pack/blob/d46d1c69b788956160deed5e4e603f4f2780ffcf/docs/src/commands/pack-and-publish.md

Reply:
It looks like it runs wasm-pack pack and the publish the content of pkg/

Not sure if that is better than our rollup bundle

Reply:
Ah. Fair. Yeah, if this works better with our rollup bundle, let's do that.